bus-mapping/src/evm/opcodes.rs

```diff
@@ -116,7 +117,7 @@ fn fn_gen_associated_ops(opcode_id: &OpcodeId) -> FnGenAssociatedOps {
         // OpcodeId::DIFFICULTY => {},
         // OpcodeId::GASLIMIT => {},
         // OpcodeId::CHAINID => {},
-        // OpcodeId::SELFBALANCE => {},
+        OpcodeId::SELFBALANCE => StackOnlyOpcode::<0>::gen_associated_ops,
```

Comment:
Shouldn't there be an CallContextOp and AccountOp READ in this opcode?  If you plan on adding those once #292 is merged, maybe you could set this PR as draft until it's ready?